woocommerce/payment-gateway/class-sv-wc-payment-gateway-my-payment-methods.php

```diff
@@ -173,6 +175,27 @@ protected function load_tokens() {
 	}
 
 
+	/**
+	 * Adds the token ID to the token data array.
+	 *
+	 * @see wc_get_account_saved_payment_methods_list
+	 *
+	 * @internal
+	 *
+	 * @since 5.6.0-dev
+	 *
+	 * @param array $item individual list item from woocommerce_saved_payment_methods_list
+	 * @param \WC_Payment_Token $token payment token associated with this method entry
+	 * @return array
+	 */
+	public function add_payment_methods_list_item_id( $item, $token ) {
+
+		$item['id'] = $token->get_token();
```

Comment:
Since this is a list of core tokens and the value represents what they call a token (instead of the ID), would it make sense to rename the entry to `token` as well? 

```suggestion
		$item['token'] = $token->get_token();
```

Reply:
I am not sure, because it is what _we_ call an `id`... Adding @ChaseWiseman for another opinion!

Reply:
I'll merge this as-is meanwhile to avoid nested PRs.

Reply:
@deivamagalhaes @wvega yeah I think naming it `token` makes sense, since at this point we're only dealing with a core token and not our wrapper of a token. This added value is also going to be available to everyone else, so it could be confusing if we add it as `id`